app/cafe.py

```diff
@@ -0,0 +1,19 @@
+import datetime
+from app.errors import NotVaccinatedError
+from app.errors import OutdatedVaccineError
+from app.errors import NotWearingMaskError
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if visitor.get("vaccine") is None:
+            raise NotVaccinatedError("Visitor is not vaccinated!")
+        else:
+            if visitor["vaccine"]["expiration_date"] < datetime.date.today():

```

Comment:
Use `elif` insted of `else` and `if` combination where possible to avoid nesting in your if statements.

Reply:
This comment is not resolved.